src/main/resources/config/liquibase/changelog/20191210131150_changelog.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet author="hashemy" id="20191210131150">
+        <createTable tableName="grading_instruction">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="credits" type="DOUBLE"/>
+            <column name="level" type="VARCHAR(255)"/>
+            <column name="instruction_description" type="LONGTEXT"/>
+            <column name="feedback" type="LONGTEXT"/>
+            <column name="usage_count" type="INT"/>
+            <column name="exercise_id" type="BIGINT"/>
+        </createTable>
+        <addForeignKeyConstraint baseColumnNames="exercise_id" baseTableName="grading_instruction" constraintName="fk_grading_instruction_exercise_id" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="exercise"/>
```

Comment:
Shouldn't the grading instruction also be deleted if the exercise itself was deleted?